tgtg/__init__.py

```diff
@@ -207,15 +247,15 @@ def set_favorite(self, item_id, is_favorite):
             raise TgtgAPIError(response.status_code, response.content)
 
     def signup_by_email(
-        self,
-        *,
-        email,
-        password,
-        name,
-        country_id="GB",
-        device_type="ANDROID",
-        newsletter_opt_in=False,
-        push_notification_opt_in=True,
+            self,
+            *,
+            email,
+            password,
+            name,
+            country_id="GB",
+            device_type="ANDROID",
+            newsletter_opt_in=False,
+            push_notification_opt_in=True,
     ):
         response = requests.post(
             self._get_url(SIGNUP_BY_EMAIL_ENDPOINT),
```

Comment:
You removed `SIGNUP_BY_EMAIL_ENDPOINT` above

Reply:
Yes, right. This is the new tgtg login endpoint.